ibu-imager/seedcreator/seedcreator.go

```diff
@@ -245,6 +251,42 @@ func (s *SeedCreator) createContainerList() error {
 	return nil
 }
 
+func (s *SeedCreator) backupSeedCertificates(ctx context.Context) error {
+	s.log.Info("Backing up seed cluster certificates for recert tool")
+	if err := os.MkdirAll(common.BackupCertsDir, os.ModePerm); err != nil {
+		return fmt.Errorf("error removing %s: %v", common.BackupCertsDir, err)
```

Comment:
```suggestion
		return fmt.Errorf("error removing %s: %v", common.BackupCertsDir, err)
``` return fmt.Errorf("error creating %s: %v", common.BackupCertsDir, err)